modules/darwin/community-builder/users.nix

```diff
@@ -118,6 +118,11 @@ let
       trusted = true;
       uid = 546;
     }
+    {
+      name = "adisbladis";
+      trusted = true;
+      uid = 547;
+    }
```

Comment:
```suggestion
```
Sorry if I wasn't clear, I meant for this to be removed and only use the hetzner user.

Reply:
Oh. That's also reasonable.